docs/swagger-1.9.json

```diff
@@ -399,6 +407,7 @@
           "user"
         ],
         "summary": "report a user with reason",
+        "description": "Notifies the related support channels for the application.",
```

Comment:
The descriptions here and above all have full stops at the end, but the ones below all don't